AuxTel/AuxTel-Troubleshooting/General-Troubleshooting/Images-out-of-focus.rst

```diff
@@ -0,0 +1,128 @@
+
+.. figure:: ./_static/out-focus.png
+    :name: out-focus
+    :width: 400
+
+
+.. _Image-out-of-focus-Procedure-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. todo::
+   Make sure everything is in a safe or idle state before Image out of focus. Describe relevant safety steps if necessary.
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+This paragraph describes some general guidance for this procedure. This paragraph is optional depending on its usefulness and the complexity of the procedure.
+
+In the case of a very straightforward procedure, a simple numbered list could be used, like as follows:
+
+.. warning::
+    For this example, this step is critical.
+
+.. _Image-out-of-focus-Procedure-Critical-Step-1:
+
+#. Pause ATScriptQueue.
+
+#. Clear all ATAOS offsets for the x, y, and z axes. Load and run the standard script `auxtel/offset_ataos` with the following configuration::
+
+    reset_offsets: all
+
+#. Run the `auxtel/latiss_wep_align` external script with the configuration below to find a new target in the area of the sky of your choice::
+
+    find_target:
+      az: 135
+      el: 40
+      mag_limit: 8.0
+
+#. Keep playing the rest of the scripts in the queue, ensuring the system is now in focus.
+
+.. _Image-out-of-focus-Procedure-Final-Step:
+
+#. Complete the procedure by verifying the system is in focus.
+
+.. _Image-out-of-focus-Procedure-Post-Condition:
+
+Post-Condition
+==============
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- System is back in focus.
+- All scripts in the queue have been successfully executed.
+
+.. figure:: ./_static/in-focus.png
```

Comment:
Considering the text in the original document it seems that this image is also out of focus in the situation "when the WEP script took the system out of focus"

Reply:
I move the image up & add better context.

- These out of focus images might range from slightly out of focus image when the WEP script took the system out of focus.